templates/todo/projects/csharp-sql-swa-func/.vscode/launch.json

```diff
@@ -0,0 +1,39 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Debug Web",
+            "request": "launch",
+            "type": "msedge",
+            "webRoot": "${workspaceFolder}/src/web/src",
+            "url": "http://localhost:3000",
+            "sourceMapPathOverrides": {
+                "webpack:///src/*": "${webRoot}/*"
+            },
+        },
+        {
+            "name": "Debug API",
```

Comment:
I was not able to get the debugger symbols loaded and my breakpoint did not hit even though the application started up fine... Does it work for you?

Reply:
Thank you, I had Phillip and Jon help me out with this one and added the change in latest commit. 
For now, due to ways VS code Azure Functions extension works, you'll have to manually start the Start API task, and then run the Debug API task (you'll be asked to pick from a list of running .NET processes, search for the name of your app and select it.) Issues have been filed to hopefully get this working as it does in other templates https://github.com/Azure/azure-dev/issues/836
https://github.com/microsoft/vscode-azurefunctions/issues/3367